src/Controller/EnMarche/Jecoute/AbstractJecouteController.php

```diff
@@ -166,7 +166,7 @@ public function jecouteSurveyStatsAction(
         SurveyExporter $exporter
     ): Response {
         if ($format = $request->query->get('export')) {
-            return $exporter->export($survey, $format, true);
+            return $exporter->export($survey, $format, false);
```

Comment:
You can not to specify `false`:
```suggestion
            return $exporter->export($survey, $format);
```

Reply:
yes but, in some cases, like that, I prefer to precise it, so we know that it is intentional and that we really want this behaviour